iroha/src/sumeragi.rs

```diff
@@ -1663,6 +1663,30 @@ pub mod config {
         }
     }
 
+    /// `SumeragiConfiguration` provides an ability to define parameters such as `BLOCK_TIME_MS`
+    /// and list of `TRUSTED_PEERS`.
+    #[derive(Clone, Debug, Deserialize)]
+    #[serde(rename_all = "UPPERCASE")]
+    pub struct TrustedPeers {
+        /// Optional list of predefined trusted peers.
+        #[serde(default)]
+        pub peers: BTreeSet<PeerId>,
+    }
+
+    impl TrustedPeers {
+        /// Load trusted peers variables from a json *pretty* formatted file.
+        pub fn from_path<P: AsRef<Path> + Debug>(path: P) -> Result<TrustedPeers> {
+            println!("{:?}", path);
+            let file = File::open(path).map_err(|e| error!("Failed to open a file: {}", e))?;
```

Comment:
It would be better to use `wrap_err` here. [`std::error::Error`](https://doc.rust-lang.org/std/error/trait.Error.html) trait has source method on it which allows natural nesting of errors inside each other. Wrapping actually does it, while helping give context of error to user,

Reply:
Also could you remove debug print from here?😅

Reply:
Thanks! I fix it 🙇